logging/config.go

```diff
@@ -118,8 +118,8 @@ func NewConfigFromMap(data map[string]string) (*Config, error) {
 }
 
 // NewConfigFromConfigMap creates a LoggingConfig from the supplied ConfigMap
```

Comment:
Update doc string to mentions `components` ?

Reply:
Done